src/main/test/task/TaskTest.java

```diff
@@ -5,7 +5,9 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 public class TaskTest {
+
     Task task = new ToDoTask("lol123");
+
     @Test
     void get_save_format_success() {
```

Comment:
Perhaps it could be better you name your test cases as per the following getSaveFormat_Success?